addons/point_of_sale/static/src/app/screens/product_screen/product_screen.xml

```diff
@@ -48,6 +34,7 @@
                                 class="pos.productViewMode"
                                 name="product.display_name"
                                 productId="product.id"
+                                color="product.pos_categ_ids?.at(-1)?.color"
```

Comment:
If one of the product's categories is the selected category, use the color of selected, otherwise, it can default to the last category.

Reply:
it's done